File: guides/adding-commit-info/java/src/main/java/example/micronaut/Application.java
package example.micronaut;

import io.micronaut.core.annotation.Introspected;
import io.micronaut.runtime.Micronaut;

@Introspected
public class Application {

    public static void main(String[] args) {
        Micronaut.run(Application.class, args);
    }
}

File: guides/adding-commit-info/kotlin/src/main/kotlin/example/micronaut/Application.kt
package example.micronaut

import io.micronaut.core.annotation.Introspected
import io.micronaut.runtime.Micronaut.*

@Introspected
class ApplicationKt {

    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            build()
                .args(*args)
                .packages("example.micronaut")
                .start()
        }
    }
}
